Lib/email/_header_value_parser.py

```diff
@@ -1360,6 +1360,9 @@ def get_word(value):
         leader, value = get_cfws(value)
     else:
         leader = None
+    if not value:
```

Comment:
Do you want to catch all false-y values or just `None`?

Reply:
I want to catch `None` and empty string `''` values, both of which could cause IndexError when running `value[0]` I suppose.